src/event.rs

```diff
@@ -1019,8 +1017,12 @@ bitflags! {
         // const RCTRL = 0b001 << 3;
         /// The "alt" key.
         const ALT = 0b100 << 6;
-        // const LALT = 0b010 << 6;
-        // const RALT = 0b001 << 6;
+        /// the left "alt" key.
+        #[cfg(target_os = "macos")]
+        const LALT = 0b010 << 6;
+        /// the right "alt" key.
+        #[cfg(target_os = "macos")]
+        const RALT = 0b001 << 6;
```

Comment:
We shouldn't change `ModifiersState` depending on the target (since that makes it quite difficult for downstream to write platform-independent code). Better to document where `LALT` and `RALT` are supported.

```suggestion
        /// the left "alt" key.
        const LALT = 0b010 << 6;
        /// the right "alt" key.
        const RALT = 0b001 << 6;
```

Reply:
Fixed